frontend/src/main/scala/bloop/engine/tasks/toolchains/ScalaJsToolchain.scala

```diff
@@ -128,20 +127,51 @@ object ScalaJsToolchain extends ToolchainCompanion[ScalaJsToolchain] {
     else sys.error(s"Expected compatible Scala.js version [0.6, 1.0], $version given")
   }
 
+  /** Determine additional version-specific artefacts */
+  private def scalaJsArtifacts(
+      platformVersion: String,
+      scalaVersion: String
+  ): List[DependencyResolution.Artifact] = {
+    // Needed for platformVersion >= 0.6.29
+    val testBridge =
```

Comment:
@sjrd Thanks for the quick reply! I was trying to add the scalajs-test-bridge artefact to the linker classpath here. Did I understand correctly that it should only be on the test runner's classpath?

Reply:
It must not be on the linker's classpath, no. The linker is a JVM thing, but the bridge is a JS thing. It must be on the classpath of the tests, as if doing
```Scala
libraryDependencies += "... bridge ..." % "test"
```
in the Scala.js project, in sbt.